src/aio.rs

```diff
@@ -219,6 +220,7 @@ where
     }
 
     /// Returns [`Stream`] of [`FromRedisValue`] values from this [`Monitor`]ing connection
+    #[allow(clippy::needless_lifetimes)]
     pub fn on_message<'a, T: FromRedisValue>(&'a mut self) -> impl Stream<Item = T> + 'a {
```

Comment:
Why not use `'_` here? (ditto fo the other places)

Reply:
I just update my MR to use `'_` instead